beekeeper-metadata-cleanup/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>beekeeper-parent</artifactId>
+    <groupId>com.expediagroup</groupId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>beekeeper-metadata-cleanup</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.expediagroup</groupId>
+      <artifactId>beekeeper-core</artifactId>
+      <version>${project.version}</version>
+      <exclusions>
+        <!-- if this is on classpath then not-null checks in objects persisted via JPA are ignored -->
+        <exclusion>
+          <groupId>org.hibernate.validator</groupId>
+          <artifactId>hibernate-validator</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>27.1-jre</version>
+    </dependency>
+  </dependencies>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <version>${springframework.boot.version}</version>
+        <configuration>
+          <classifier>spring-boot</classifier>
+          <mainClass>com.expediagroup.beekeeper.metadata.cleanup.BeekeeperMetadataCleanup</mainClass>
```

Comment:
Opinions on whether the base package name should be `com.expediagroup.beekeeper.metadata.cleanup` versus `com.expediagroup.beekeeper.cleanup.metadata`? The latter would make it more like a sub-package of [beekeeper-cleanup](https://github.com/ExpediaGroup/beekeeper/tree/master/beekeeper-cleanup) which may or may not be what you want. I guess it depends whether the focus here is "metadata" or "cleanup".

Reply:
I think focus is more on metadata although the waters are a bit muddy as we'll probably delete data from this module as well yesterday we discussed a shared module that hold the "deletion" logic. 
The features are broadly split into
Unreferencing data: cleaning up paths only (The old cleanup)
Expiring data: cleaning up metadata (tables/partitions) + corresponding paths (the new metadata cleanup)